sources/views/top.js

```diff
@@ -45,6 +52,20 @@ export default class TopView extends JetView {
 	}
 
 	init() {
+		this.topMenu = this.$$("top:menu");
 		this.use(plugins.Menu, "top:menu");
+
+		this.topMenu.attachEvent("onAfterSelect", (id) => {
+			if (id) {
+				const menuItem = this.topMenu.getItem(id).value;
+				this.$$("topHeader").setValues({menuItem});
+			}
+		});
+	}
+
+	urlChange(view, url) {
```

Comment:
No need to call this every time. If you plan to restore the opened view after the page was reloaded, do this once (on init, for example)

Reply:
fixed

Reply:
I didn't notice that you use `this.use(plugins.Menu, "top:menu");` - my bad.
This urlChange handler can be removed at all, as Menu plugin restores the selection.

Reply:
fixed